docs/docs/API-Reference/api-flows-run.md

```diff
@@ -27,6 +27,7 @@ The parameters are passed in the request body. In this example, the values are t
 curl -X POST \
```

Comment:
I just realized there are some instances of `x-api-key` outside of the curl examples that need to edited or deleted in this PR:

* webhook.md ~line 29
* api-flows-run.md in "Run endpoint headers" section: This section also says "accept" header is required, but this header isn't included in the auto-generated code snippets. (asked about this in slack too)
* api-build.md ~line 95 & 96: Same concerns as with api-flows-run.md regarding x-api-key and accepts headers.

Also, mcp-server.md includes `x-api-key` in various places, but this PR says those endpoints don't require auth?

Reply:
MCP is only if you have auth enabled - the copy snippets don't include it
<img width="857" alt="Screenshot 2025-07-07 at 11 22 15 AM" src="https://github.com/user-attachments/assets/1f1f6afc-d828-41f4-96ae-96ba8006e93a" />

Reply:
accept is optional (confirmed in Slack).